index.js

```diff
@@ -210,6 +211,30 @@ module.exports = function(version, _options) {
             }
 
             return output;
+        },
+        findAppropriateLanguage: function(language) {
+            if (languages.supportedCodes.indexOf(language) > -1) {
```

Comment:
`languages.instructions` is an object with language codes as the keys. I think it’d be a bit easier to work with that object than the supported codes array.

Reply:
Let's keep the list of languages we're using here consistent. An array of locales is little more accurate than a dictionary with keys IMO.